packages/datagovmy-ui/src/data-catalogue/Show/metadata.tsx

```diff
@@ -43,6 +45,21 @@ const DCMetadata: FunctionComponent<MetadataProps> = ({
   const { t, i18n } = useTranslation(["catalogue", "common"]);
   const { dataset } = useContext(CatalogueContext);
   const { track } = useAnalytics(dataset);
+  const [selectedEdition, setSelectedEdition] = useState<OptionType | undefined>(
+    metadata.link_editions && metadata.link_editions.length > 0
+      ? { label: metadata.link_editions[0], value: metadata.link_editions[0] }
+      : undefined
+  );
+
+  useEffect(() => {
```

Comment:
help me understand what this useEffect is for?

Reply:
sorry my bad here, i've already set the initial state in the useState, so this useEffect is redundant, i should remove it